spec/workers/email_worker_spec.rb

```diff
@@ -0,0 +1,4 @@
+require 'rails_helper'
+RSpec.describe EmailWorker, type: :worker do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
Either add some tests or remove this file. I'd say just remove the file so you don't stay too long in this card